.github/workflows/bot.yml

```diff
@@ -148,3 +148,43 @@ jobs:
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
           commit_message: "style: apply lint fixes"
+
+  Check_Degdratation:
+    name: Criterion Benchmarks
```

Comment:
There is no iai-callgrind benchmarks anymore?

Reply:
i am not running iai-callgrind as criterion is better for regression checks , but i have kept iai-callgrind files as we might need to see small change in performances in future

Reply:
well, there is no better runner, they are just different.

we still want to have both kind of benchmarks that shows different representation of performance and the way to see the performance regression in both terms. See the [comment](https://github.com/tailcallhq/tailcall/issues/436#issuecomment-1794164324) from the original issue.

As I remember you already had implementation for this and it just was named mistakenly `Build time`, so we can bring it back probably.

To summarize, we want:
- criterion benchmarks
- iai-callgrind benchmarks

both as separate ci job and different script to handle